content/departments/cloud/index.md

```diff
@@ -149,7 +149,7 @@ Support SLAs for Sev 1 and Sev 2 can be found [here](../ce-support/support/index
 | Existing instance suspension                                                      | Suspend an existing managed instance temporarily       | Within 24 hours of becoming aware of the need                                                                                     | Within 15 working days from agreement                                                                                             |
 | Existing instance deletion/teardown                                               | Decommission/delete and existing managed instance      | Within 24 hours of becoming aware of the need                                                                                     | Within 15 working days from agreement                                                                                             |
 | New Feature Request                                                               | Feature request from new or existing customers         | Within 24 hours of becoming aware of the need                                                                                     | Dependent on the request                                                                                                          |
-| Maintenance: Monthly Update to latest release                                     | Updating an instance to the latest release             | NA                                                                                                                                | Within 1 week after latest release                                                                                                |
+| Maintenance: Monthly Update to latest release                                     | Updating an instance to the latest release             | NA                                                                                                                                | Within 10 working days after latest release                                                                                       |
```

Comment:
@RafLeszczynski should we explain +6 days here when patch occurs? or they are handled by next row in this table sufficient? (`patch/emergency release`)